spec/03-plugins/26-prometheus/04-status_api_spec.lua

```diff
@@ -232,7 +232,7 @@ describe("Plugin: prometheus (access via status API)", function()
         path    = "/metrics",
       })
       local body = assert.res_status(200, res)
-      return body:find('kong_http_status{service="mock-grpcs-service",route="grpcs-route",code="200"} 1', nil, true)
+      return body:find('kong_http_status{service="mock-grpcs-service",route="grpcs-route",code="200",service_tags="",route_tags=""} 1', nil, true)
```

Comment:
Please add a test case for when there are tags on the service or route. Please make sure to have multiple tags to test for sort order.